### DISCUSSION TITLE
[SUGGESTION] Mimic actuators

### DISCUSSION THREAD

**Discussion by cloudrambler** [tsunamayo/Starship-EVO#4033] [Suggestions]
Mechanisms often have duplicate actuators for aesthetic reasons; parallel pistons, or opposite rotors for example. I have noticed that firing many actuators at once, such as for landing gear, causes some of them to lag. This is understandable, but I wonder is it also avoidable.

My suggestion is that rather than driving all the actuators separately, we have a method to directly link them together so that they all act identically. This could be done by using the link tool from one actuator to another that is mounted in the same plane or axis (as applicable). That would then allow for their outputs to be tied together, reducing the number of collision checks to perform for that mechanism.

**Comment by nokturnihs** [comment]
I like this idea or something like it! Could also be used to accommodate/increase higher mass limits on certain mechanisms. Don't  know how you'd do that with rotors except to allow the rotor to scale to accommodate higher mass limits but I love this idea for lifts, landing gears, slide rails, hinges, pistons, etc. By linking you could also scale power consumption if that's somewhere you're interested in going in that regard...
